kod/object/passive/ornobj.kod

```diff
@@ -1286,6 +1448,14 @@ messages:
       {
          send(oDuplicate,@SetNoMoveOn,#value=TRUE);
       }
+      if Send(self, @GetObjectFlags) & FLICKERING_YES
+      {
+         Send(oDuplicate, @SetFlickering, #value=TRUE);
+      }
+      if NOT Send(self, @GetObjectFlags) & LOOK_NO
+      {
+         Send(oDuplicate, @SetLookable, #value=TRUE);
+      }
```

Comment:
I wasn't too sure about even touching this function since it seems like only rented rooms use it, but I can see the value for admins who might want to make, say, multiple signs with the same special description or something for different inns/etc to advertise an event, so added custom description copying.

I'm unsure about timers though since all it would do is copy the other object's remaining time to make its own timer, and I'm not sure how valuable that'd even be.  It's easy enough for me to add though if you think it should be there.